webapp-django/crashstats/crashstats/views.py

```diff
@@ -91,14 +88,11 @@ def report_index(request, crash_id, default_context=None):
 
     raw_api = models.RawCrash()
     try:
-        context['raw'] = raw_api.get(
-            crash_id=crash_id,
-            refresh_cache=refresh_cache,
-        )
+        context['raw'] = raw_api.get(crash_id=crash_id, refresh_cache=refresh_cache)
     except CrashIDNotFound:
         # If the raw crash can't be found, we can't do much.
-        tmpl = 'crashstats/report_index_not_found.html'
-        return render(request, tmpl, context, status=404)
+        return render(request, 'crashstats/report_index_not_found.html', context, status=404)
+    utils.enhance_raw(context['raw'])
```

Comment:
This is the new line. The rest of the changes are reducing vertical space usage and are cosmetic.